substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/NativeLibraries.java

```diff
@@ -146,6 +146,7 @@ private boolean loadLibrary0(File file, boolean builtin) {
             String canonical = builtin ? file.getName() : file.getCanonicalPath();
             return addLibrary(canonical, builtin);
         } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
We don't know where the library is supposed to be and this method is used for probing, so adding this might print several unhelpful stack traces even when the library can ultimately be loaded. I believe the behavior is the same in OpenJDK, see `jdk.internal.loader.NativeLibraries#loadLibrary(java.lang.Class<?>, java.io.File)`.

Reply:
@peter-hofer At least print it with debug level if possible instead of hiding the exception details entirely.